Makefile

```diff
@@ -0,0 +1,29 @@
+prepare:
+	rustup target add wasm32-unknown-unknown
+
+build-contract:
+	cargo build --release -p cask --target wasm32-unknown-unknown
+
+test-only:
+	cargo test --workspace
+
+copy-wasm-file-to-test:
+	mkdir -p tests/wasm
+	cp target/wasm32-unknown-unknown/release/*.wasm tests/wasm
+
+test: build-contract copy-wasm-file-to-test test-only
+
+clippy:
+	cargo clippy --all-targets --all -- -D warnings -A renamed_and_removed_lints
```

Comment:
we should really stop disabling any of the warnings and errors on linting

Reply:
Is there an official CL set of lints we should apply to contracts? My experience just running clippy out of the box (rather than just relying on CLion to tell me I'm wrong) resulted in it complaining about things that would break when "fixed."

Reply:
In https://github.com/rust-lang/rust-clippy#travis-ci, it says

> ...
> Note that adding -D warnings will cause your build to fail if any warnings are found in your code. That includes warnings found by rustc (e.g. dead_code, etc.).
> ...